src/set_module.f90

```diff
@@ -2146,13 +2159,18 @@ subroutine set_metadyn(env,key,val)
    logical,save :: set2 = .true.
    logical,save :: set3 = .true.
    logical,save :: set4 = .true.
+   logical,save :: set5 = .true.
 
    select case(key)
    case default ! do nothing
       call env%warning("the key '"//key//"' is not recognized by metadyn",source)
    case('save')
       if (getValue(env,val,idum).and.set1) metaset%maxsave = idum
       set1 = .false.
+   case('dump')
+      call set_siman(env,'dump',val)
+      if (getValue(env,val,ddum).and.set5) dump_md = ddum
+      set5 = .false.   
```

Comment:
This is probably not a good idea, as it allows to overwrite `dump_md` even if it should be locked already through `set_md`. Let's remove this part and also remove the `set_siman` completely, such that it is only set in the `set_md` block.